### ISSUE TITLE
Implement Couchbase adapter (in separated repository)

### ISSUE THREAD

**Issue by cabol** [cabol/cross_db#4]

**Comment by hengestone** [comment]
Fwiw, the erlcass (datastax nif) works very well here, using yugabyte-db as well as Scylla.

**Comment by cabol** [comment]
Nice! But I think those libs applies to the Cassandra or Scylla adapter right? I've just created an issue for [Cassandra adapter](https://github.com/cabol/cross_db/issues/5), which is in the short term road map for sure.

**Comment by hengestone** [comment]
Yes, both Yugabyte and Scylla are Cassandra protocol compatible. I mentioned the erlcass driver because in comparison marina has issues with Yugabyte, and cqerl is quite a bit slower.

**Comment by cabol** [comment]
Yes agree, I was checking `erlcass` and looks good, it might be a very good option for Cassandra adapter.
